ddp-workspace/projects/ddp-sdk/src/lib/services/sessionMemento.service.ts

```diff
@@ -8,7 +8,7 @@ import { filter, map, mergeMap, startWith } from 'rxjs/operators';
 export class SessionMementoService implements OnDestroy {
     private readonly SESSION_KEY: string = 'session_key';
     private readonly TOKEN_KEY = 'token';
-    private readonly MSECS_TO_NOTIFICATION = 300000;
+    private readonly MSECS_TO_NOTIFICATION = 360000;
```

Comment:
Would it help to add more "leeway" to the [token expiration server-side](https://www.javadoc.io/doc/com.auth0/java-jwt/3.0.1/com/auth0/jwt/JWTVerifier.Verification.html#acceptLeeway-long-)?  I think we're currently at 10 seconds.

Reply:
I think there should be enough changes only on the frontend side.